react/__mocks__/vtex.render-runtime.tsx

```diff
@@ -1,3 +1,11 @@
 import React, { FunctionComponent } from 'react'
 
 export const Loading: FunctionComponent = () => <div></div>
+
+export const useRuntime = () => {
+  let rootPath = ''
+  const setRootPath = (rP: string) => {
+    rootPath = rP
+  }
```

Comment:
we can use the other hooks here too, although you don't need to return the `setRootPath` function
```suggestion
  const [rootPath, setRootPath] = useState('')
```

Reply:
Ready, thanks!